### PR TITLE
Refactor sample DB and add config files for Jest, Enzyme, and CircleCI

### PR DESCRIPTION
**Pull request by iamjaytong**
Made some updates from prior PR re: checking database connection and tidying up the Accounts schema. Added the .gitignore file back in along with all the config files for Jest, Enzyme, CircleCI. Jest & Enzyme already confirmed working on local, will use this PR to see if CircleCI works with the sample code and test. 

See sample code and test in:
src/WelcomeMessage.js
src/WelcomeMessage.test.js

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:8.15.1
+
+    steps:
+      - checkout
+      - run: npm install
+      - run: npm test
```
Comment (vh02) [inline .circleci/config.yml]: Remember airBnB style guide, adding a return at the end of the file!

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_modules
+.DS_Store
```
Comment (vh02) [inline .gitignore]: Awesome

File: database-mysqldb/Accounts.js [hunk 3]
```diff
@@ -8,15 +8,9 @@ const Account = db.define('account', {
...
-  },
-  option_level: {
-    type: Sequelize.INTEGER,
-  },
-  watchlist: {
-    type: Sequelize.STRING,
-  },
+  buying_power: Sequelize.DECIMAL(14, 4),
+  option_level: Sequelize.INTEGER,
+  watchlist: Sequelize.STRING,
```
Comment (vh02) [inline database-mysqldb/Accounts.js]: If we refer back to your original seed for Accounts.js, maybe we can consider generating the 100 stocks dynamically, versus hard-coding it?

Comment (vh02) [reply database-mysqldb/Accounts.js]: Also, in seed.js again, why did you use different types to represent buying_power vs quantity? (and choosing to use underscores in certain variable names over others). Are you trying to address floating point precision?

Comment (vh02) [reply database-mysqldb/Accounts.js]: The Sequelize.DECIMAL(14, 4) vs. the Sequelize.INTEGER made me take a second look

Comment (iamjaytong) [reply database-mysqldb/Accounts.js]: Yep good catch - DECIMAL is specifically used here to address the floating point precision issue.

File: src/WelcomeMessage.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    marginBottom: 15,
+  },
+  logo: {
+    width: 250,
+    heading: 250,
+    objectFit: 'cover',
+  },
+};
+
+const WelcomeMessage = ({ imgPath }) => {
```
Comment (vh02) [inline src/WelcomeMessage.js]: The imgPath is to be defined?

File: src/WelcomeMessage.test.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+// Components
+import WelcomeMessage from './WelcomeMessage';
+
+function setup() {
+  const props = {
+    imgPath: 'some/image/path/to/a/mock/image',
```
Comment (vh02) [inline src/WelcomeMessage.test.js]: /buysell_module/dist/lib/ could be a good place to store images

**Pull request merged**